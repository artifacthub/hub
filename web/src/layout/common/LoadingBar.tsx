import React from 'react';

import styles from './LoadingBar.module.css';

const LoadingBar = () => (
  <div className={`position-fixed ${styles.wrapper}`}>
    <div className={`${styles.progressLine} progressLine`} role="status" />
  </div>
);

export default LoadingBar;
