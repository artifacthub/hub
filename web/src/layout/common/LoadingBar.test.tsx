import { render } from '@testing-library/react';
import React from 'react';

import LoadingBar from './LoadingBar';

describe('LoadingBar', () => {
  it('creates snapshot', () => {
    const { asFragment } = render(<LoadingBar />);
    expect(asFragment).toMatchSnapshot();
  });
});
