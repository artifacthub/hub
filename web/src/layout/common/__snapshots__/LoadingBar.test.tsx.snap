// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingBar creates snapshot 1`] = `[Function]`;
